test/.eslintrc

```diff
@@ -0,0 +1,6 @@
+{
+  "rules": {
+    "func-names": 0,
+    "import/no-extraneous-dependencies": 0
```

Comment:
This is added so we can have
```
function () { ... }
```
syntax in the tests, and to be able to have `devDependencies`.